script.recap/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.recap" name="Recap" version="1.0.0" provider-name="robwebset">
+	<requires>
+		<import addon="xbmc.python" version="2.24.0"/>
+		<import addon="script.module.beautifulsoup" version="3.2.1"/>
+		<import addon="script.module.requests" version="2.3.0"/>
+	</requires>
+  	<extension point="xbmc.python.script" library="default.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="kodi.context.item">
+		<menu id="kodi.core.main">
+			<item library="default.py">
+				<label>32001</label>
+				<visible>IsEmpty(Window(videos).Property("RecapHideContextMenu")) + [StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,season) | StringCompare(ListItem.dbtype,episode)]</visible>
```

Comment:
It may just be me, but I don't like the idea of add-ons adding context menu items by default, and I was assuming (which I shouldn't have done) this one didn't. I understand the reasoning and have been tempted myself. It doesn't seem bad in the context of a single addon, but eventually we can run into a situation where users have a context menu full of items which they may not want, and are not sure which installed addon provided it. This is especially true with add-ons that require the context menu addon or (as is the case here) include it in the same add-on.

@phil65: Thoughts? I'm fine with it otherwise, as the extension points are not doing anything wrong themselves, and the service is not a long running service. If he breaks the context menu out as an addon, that doesn't solve the issue, because he could just add it as a dependency, and then we're in basically the same position (unless we're not allowing context menu items as a dependency - and if that's the case then the point is moot).